tools/jbrowse2/jbrowse2.xml

```diff
@@ -0,0 +1,873 @@
+                                <param name="match_part_select" value="false"/>
+                            </conditional>
+                            <section name="jbcolor_scale">
+                                <conditional name="color_score">
+                                    <param name="color_score_select" value="none"/>
+                                </conditional>
+                                <conditional name="color">
+                                    <param name="color_select" value="automatic"/>
+                                </conditional>
+                            </section>
+                        </conditional>
+                    </repeat>
+                    <repeat name="data_tracks">
+                        <conditional name="data_format">
+                            <param name="data_format_select" value="gff"/>
+                            <conditional name="useuri">
+                                <param name="annotation" value="gff3/2.gff"/>
+                                <param name="insource" value="history"/>
+                            </conditional>
+                            <conditional name="match_part">
+                                <param name="match_part_select" value="true"/>
+                                <param name="name" value="cDNA_match"/>
+                            </conditional>
+                            <section name="jbcolor_scale">
+                                <conditional name="color_score">
+                                    <param name="color_score_select" value="none"/>
+                                </conditional>
+                                <conditional name="color">
+                                    <param name="color_select" value="automatic"/>
+                                </conditional>
+                            </section>
+                        </conditional>
+                    </repeat>
+                </repeat>
+            </repeat>
+            <param name="uglyTestingHack" value="enabled"/>
+            <output name="output">
+                <assert_contents>
+                    <has_text text="Ignore Scale"/>
+                    <has_text text="A.gff"/>
+                    <has_text text="B.gff"/>
+                    <has_text text="C.gff"/>
+                    <has_text text="interpro.gff"/>
+                    <has_text text="Scaled Colour"/>
+                    <has_text text="1.gff"/>
+                    <has_text text="2.gff"/>
+                </assert_contents>
+            </output>
+        </test>
+        <!-- TODO add a synteny test -->
+        <!-- TODO add a bam and a cram test -->
+        <!-- TODO add an hic test -->
+        <!-- TODO add a vcf_bgzip test -->
+    </tests>
+    <help><![CDATA[
+
+JBrowse2-in-Galaxy
+==================
+
+JBrowse2-in-Galaxy offers a highly configurable, workflow-compatible
+alternative to JBrowse1-in-Galaxy and Trackster.
```

Comment:
```suggestion
JBrowse2-in-Galaxy offers a highly configurable, workflow-compatible Genome viewer.
```

Trackster is gone I think, or will be gone.

Reply:
Yes please. Didn't know it had disappeared but about time.